WooCommerce/UITestsFoundation/Screens/Orders/OrdersScreen.swift

```diff
@@ -17,17 +26,20 @@ public final class OrdersScreen: ScreenObject {
         (try? OrdersScreen().isLoaded) ?? false
     }
 
-    init(app: XCUIApplication = XCUIApplication()) throws {
+    public init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
             expectedElementGetters: [
                 searchButtonGetter,
-                // swiftlint:disable:next opening_braces
-                { $0.buttons["order-filter-button"] }
+                filterButtonGetter
             ],
             app: app
         )
     }
 
+    static func isLoaded() -> Bool {
+        return XCUIApplication().buttons[ElementStringIDs.searchButton].exists
+    }
+
```

Comment:
The `SceenObject` class, which `OrdersScreen` instantiates, already has an `isLoaded` method. That method will check for the existence of the first element from `expectedElementGetters`.  So, current method is not necessary.

```suggestion
```

Reply:
Ah ok, I've removed this (and all the references) and it does work. This has been reverted in the latest commit.